src/Microsoft.ApplicationInsights.AspNet/SdkVersionUtils.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Microsoft.ApplicationInsights.AspNet
+{ 
+    using System.Linq;
+    using System.Reflection;
+
+    internal class SdkVersionUtils
```

Comment:
I just believe in this pattern. What can I do to convince that it is good? Imagine - when you doing TrackMetric you can distinct whether it is call to TrackMetric or PerformanceCountersCollection module or @lukekim 's metric aggregation module. Will it be cool?